codice/alliance
# CAL-346 Added documentation for NITF mappings

#### What does this PR do?
This PR adds documentation for NITF mappings.

**NOTE**: Any concerns about corrections to current mappings should be addressed in new Alliance tickets.
#### Who is reviewing it? 
@lcrosenbu @peterhuffer @AzGoalie 
#### Select relevant component teams:
@codice/docs @ricklarsen 
@codice/data
#### Choose 2 committers to review/merge the PR.
@bdeining
@clockard
#### How should this be tested?

- Build docs.
- Review new content and check that it matches the `imaging-transformer-nitf` code.
- Check for typos.
- Confirm formatting (especially new lines in the `Translation` column) looks correct in the pdf and the html Documentation ([attached](https://github.com/codice/alliance/files/1345101/documentation.zip)).


#### Any background context you want to provide?
#### What are the relevant tickets?
[CAL-346](https://codice.atlassian.net/browse/CAL-346)
#### Screenshots
![image](https://user-images.githubusercontent.com/8041246/31027068-b97efdac-a4fd-11e7-91df-f190249cb4bd.png)
#### Checklist:
